test/parallel/test-os.js

```diff
@@ -157,13 +157,13 @@ if (common.isWindows) {
 } else {
   is.number(pwd.uid);
   is.number(pwd.gid);
-  assert.notStrictEqual(pwd.shell.indexOf(path.sep), -1);
+  assert.notStrictEqual(pwd.shell.includes(path.sep), false);
```

Comment:
This is fine the way it is, but it could probably just be something like `assert.ok(!pwdshell.includes(path.sep));` if you want.